.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+**Link the Issue(s) this Pull Request is related to.**
+
+Each PR should link at least one issue, in the form:
+
+```Fixes #123```
```

Comment:
```suggestion
Fixes #123
```
I guess that was copied from OTIO repo. Using triple backquotes here will lead to `Fixes #123` which GitHub won't be able to understand and so won't link the PR to the issue. Eventually it also needs to be fixed in OTIO.

Reply:
yes, this was just copied from the main OTIO repo.